### PR TITLE
Add Harvester v1.0.0-rc1 release

### PR DESCRIPTION
**Pull request by guangbochen**
Add Harvester v1.0.0-rc1 release

- [x] tag harvester api and harvester-webhook to `v1.0.0-rc1`
- [x] tag harvester-installer to [v0.2.7](https://github.com/harvester/harvester-installer/releases) 
- [x] tag harvester-dashboard to [v1.0.0-rc1](https://github.com/harvester/dashboard/releases/tag/v1.0.0-rc1)
- [x] tag harvester-network-controller to [v0.2.5](https://github.com/harvester/network-controller-harvester/releases)
- [x] tag harvester-node-disk-managet to [v0.3.2](https://github.com/harvester/node-disk-manager/releases/tag/v0.3.2)
- [x] Longhorn already bumped to v1.2.3-rc1, Kubevirt has no version bump required
- [x] tag harvester-load-balancer to [v0.1.1](https://github.com/harvester/load-balancer-harvester/releases)

**v1.0.0-rc1 tag commit:** https://github.com/harvester/harvester/pull/1605/commits/e4f43d7482c187c2b599f945dc8332eea5343694

### CODE REVIEW

**Comment by gitlawr** [comment]
We are still using `rancher/harvester-load-balancer:master-head`

**Comment by guangbochen** [comment]
> We are still using `rancher/harvester-load-balancer:master-head`

Nope, will fix it with a specific version[done]

File: deploy/charts/harvester/values.yaml [hunk 1]
```diff
@@ -418,7 +420,7 @@ harvester-load-balancer:
   image:
     imagePullPolicy: Always
```
Comment (gitlawr) [inline deploy/charts/harvester/values.yaml]: We don't overwrite the imagePullPolicy in https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rancherd-10-harvester.yaml#L119-L120
We need to set it to IfNotPresent

Comment (guangbochen) [reply deploy/charts/harvester/values.yaml]: resolved by https://github.com/harvester/harvester/compare/fc4876635550550c7119ad07b1cb8c71c876130a..5643f680beb90ae6a52bf4d9c536bd1f7880bc1e, good catch, thanks.

**Review by bk201 — APPROVED** [review]
Some manual checks:
- [x] Air-gapped env works
- [x] Grub menu displays correct versions (both livecd and after-installed)
- [x] Correct versions record in /etc/harvester-release.yaml
- [x] Able to provision VMs

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**